### PR TITLE
[MOD] Extract javascript of compose page to appropriate block

### PR DESCRIPTION
**Pull request by Th3Mouk**
for 2.3.10

### CODE REVIEW

File: Resources/views/PageAdmin/compose.html.twig [hunk 1]
```diff
@@ -82,25 +82,27 @@ file that was distributed with this source code.
...
-                    window.SonataPageComposer = composer;
-                });
-            </script>
         {% endif %}
     </div>
+{% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+    <script>
```
Comment (Soullivaneuh) [inline Resources/views/PageAdmin/compose.html.twig]: Missing `type="text/javascript"`.

**Comment by Th3Mouk** [comment]
This PR is indirectly link to #528 and allow javascripts customization without managing html part

**Comment by Soullivaneuh** [comment]
Not sure but could be BC break. ping @rande

**Comment by OskarStark** [comment]
@rande can you please give us some feedback on this PR?

**Comment by Th3Mouk** [comment]
@rande can i have your opinion ?

**Comment by core23** [comment]
IMO the modern way is to remove all inline javascript from the page and create a data listener for it.

Something like

```html
<!-- template.html -->
<div data-sonata-composer='{"translation": ... }'>
```

```js
// composer.js
    $(function () {
        $('[data-sonata-composer]').each(function () {
            new PageComposer(...);
        });
    });
```

**Comment by Th3Mouk** [comment]
Probably but the goal here is just to extract javascript to a block

**Comment by Th3Mouk** [comment]
ping @rande

**Comment by OskarStark** [comment]
change itself looks ok to me

to move all inline js to separate files should be done in a separate PR IMO

**Comment by core23** [comment]
This PR could not get merged into master,  because we use the mentioned data listener.

**Comment by Th3Mouk** [comment]
@core23 this PR is for 2.3 branch because i'm not working on unstable master

**Comment by OskarStark** [comment]
@core23 could you provide a solution for master and we merge this PR without cherry picking into 2.3?

**Comment by OskarStark** [comment]
ping @rande @Soullivaneuh

**Comment by OskarStark** [comment]
any news on this PR?

**Comment by core23** [comment]
Think we can merge this for the 2.3 branch. The master got a better solution for this https://github.com/sonata-project/SonataPageBundle/pull/562

**Comment by OskarStark** [comment]
no possibility to backport #562 to `2.3` ?

**Comment by core23** [comment]
I wouldn't do a backport for that, because it's a bigger change in the code.

**Comment by OskarStark** [comment]
:+1:

**Comment by OskarStark** [comment]
Thank you @Th3Mouk

**Pull request merged**